SDWebImage/Core/SDImageCache.h

```diff
@@ -177,7 +177,7 @@ typedef NS_OPTIONS(NSUInteger, SDImageCacheOptions) {
 - (void)storeImage:(nullable UIImage *)image
             forKey:(nullable NSString *)key
             toDisk:(BOOL)toDisk
-        completion:(nullable SDWebImageNoParamsBlock)completionBlock;
+        completion:(nullable SDWebImageNoParamsBlock)completionBlock NS_SWIFT_NAME(store(image:key:disk:completion:));
```

Comment:
`disk` doesn't sound like a boolean. Maybe `shouldStoreToDisk`?

Reply:
Actually I wanna simplify swift version parameters name and length. About this parameter do u have any other advice.

Reply:
https://swift.org/documentation/api-design-guidelines/

> Clarity is more important than brevity
`disk` is short but not clear. `shouldStoreToDisk` is clear.

> Uses of Boolean methods and properties should read as assertions about the receiver when the use is nonmutating, e.g. x.isEmpty, line1.intersects(line2)
`disk` is not read as assertion while `shouldStoreToDisk` is.